bench/wtperf/wtperf_run_py/perf_stat.py

```diff
@@ -74,8 +96,56 @@ def get_value(self):
         min_3_vals = sorted(self.values)[:3]
         return self.average(min_3_vals)
 
+
 class PerfStatMax(PerfStat):
     def get_value(self):
         """Return the averaged maximum of all gathered values"""
         max_3_vals = sorted(self.values)[-3:]
         return self.average(max_3_vals)
+
+
+class PerfStatCount(PerfStat):
+    def find_stat(self, test_stat_path: str):
+        """Return the total number of times a pattern matched"""
+        total = 0
+        test_stat_path = glob.glob(test_stat_path)[0]
+        for line in open(test_stat_path):
+            match = re.search(self.pattern, line)
+            if match:
+                total += 1
+        return [total]
+
+
+class PerfStatLatency(PerfStat):
+    def __init__(self,
+                 short_label: str,
+                 stat_file:str,
+                 output_label: str,
+                 num_max: int):
+                 super().__init__(short_label=short_label,
+                                  stat_file=stat_file,
+                                  output_label=output_label)
+                 self.num_max = num_max
+
+    def find_stat(self, test_stat_path: str):
+        values = []
+        if os.path.isfile(test_stat_path):
+            for line in open(test_stat_path):
+                as_dict = json.loads(line)
+                values.append(as_dict["wtperf"]["read"]["max latency"])
+                values.append(as_dict["wtperf"]["update"]["max latency"])
+        return values
+
+    def get_value(self, nth_max: int):
+        return sorted(self.values)[-nth_max]
```

Comment:
Can you add a comment explaining what is being returned like in the other `get_value` functions?

Reply:
yup will do! 👍